ubuntu/standard/Dockerfile.template

```diff
@@ -80,6 +80,15 @@ RUN curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /et
 # Run dotnet command to populate initial cache
 RUN dotnet help
 
+# Configure AzCLI 2.0  Instructions taken from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli
+RUN echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | \
+     tee /etc/apt/sources.list.d/azure-cli.list \
```

Comment:
Do we know that "wheezy" as a debian version identifier is going to be correct across potentially multiple Ubuntu versions (16.04 only for now, but may be newer versions in the future)? The .NET Core SDK installation instructions expect you to pass an Ubuntu version identifier instead (see line 74).

Reply:
Also, please line up the indentation of the "echo" and "tee" commands.

Reply:
Currently only the wheezy version identifier is available with azure-cli source. In their documentation, they are not specifying any version specific install steps. I tried to run this on Ubuntu 17.04 and it works - but I am not sure if they will keep this up-to-date with future Ubuntu versions. I think we might have to work with them in such cases when we add support for more Ubuntu versions. I am not sure that we should ask AzCli team to create different version identifiers in their source even when the same step can work today.